### PR TITLE
Mobx charts - victory charts + colors + chart item selector

### PR DESCRIPTION
**Pull request by na9da**
Consolidated PR for the following changes:
- Chart implementation using victory charts
- Chart colors
- Chart item selector in workbench

### CODE REVIEW

File: lib/ModelMixins/TableMixin.ts [hunk 1]
```diff
@@ -149,6 +151,83 @@ export default function TableMixin<T extends Constructor<Model<TableTraits>>>(
       ]);
     }
 
+    /**
+     * Gets the items to show on a chart.
+     *
+     * TODO: rename to chartItems() when we have removed all references to the
+     * current implementation.
```
Comment (tephenavies) [inline lib/ModelMixins/TableMixin.ts]: What's the plan for returning everything to use `chartItems` again?

**Comment by AnaBelgun** [comment]
reminder @steve9164 to review this one (needed for release of NSW DT for Wed the latest)

File: lib/ReactViews/Custom/Chart/ChartPanel.jsx [hunk 2]
```diff
@@ -1,137 +1,113 @@
...
-      //     if (!defined(xUnits) && defined(item.xAxis)) {
-      //       xUnits = item.xAxis.units;
-      //     }
-      //   }
-      // });
+  chartableItems() {
+    return this.props.terria.workbench.items
+      .filter(c => Chartable.is(c) && c.chartItems2.length > 0)
+      .filter(c => c.showInChartPanel);
+  }
```
Comment (tephenavies) [inline lib/ReactViews/Custom/Chart/ChartPanel.jsx]: Isn't this already in ChartView? Could you use that one?

File: lib/ModelMixins/TableMixin.ts (hunk 1 above)
Comment (na9da) [reply lib/ModelMixins/TableMixin.ts]: The current `chartItems()` method is called from the old chart stack (mostly `ChartRenderer`) which will become redundant once we merge this PR.  Once we are ok to merge this PR, I'll remove `ChartRenderer` and  friends and  rename `chartItems2 -> chartItems`.

**Review by tephenavies — APPROVED** [review]

**Pull request merged**